src/figuresClasses.ts

```diff
@@ -1,11 +1,78 @@
-export interface Figure {}
+export interface Figure {
+  shape: string;
+  color: string;
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  shape: string = 'triangle';
+
+  constructor(
+    public color: string,
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    if (a <= 0 || b <= 0 || c <= 0) {
+      throw new Error('Side lengths must be greater than 0');
+    }
+
+    const sideSorted: number[] = [a, b, c].sort(
+      (side1, side2) => side2 - side1,
+    );
+
+    if (sideSorted[0] >= sideSorted[1] + sideSorted[2]) {
```

Comment:
The triangle inequality check is correct, but ensure that `a`, `b`, and `c` are accessible within the class. Consider declaring them as class properties.